UPGRADE.md

```diff
@@ -2474,6 +2474,84 @@ To support utf8mb4 we needed to change some database entities which you also nee
 Run the following SQL to migrate to the new schema:
 
 ```sql
+ALTER TABLE ca_categories CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```

Comment:
@danrot depending on the mysql version this could throw an error todo this before shorten the indexed fields.

Reply:
Are you sure? The [guide I based this upon](https://mathiasbynens.be/notes/mysql-utf8mb4) seems pretty complete, and is also explaining it this way. Would be even more afraid that it might not work the other way around :see_no_evil: Have you ever seen this failing?

Reply:
It depends on the mysql version. `5.6.21` for example will throw an error in this case.